ui/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "chaos_monkey",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "bootstrap": "^4.1.3",
+    "mobx": "^5.5.1",
+    "mobx-react": "^5.3.4",
+    "moment": "^2.22.2",
+    "react": "^16.5.2",
+    "react-bootstrap": "^1.0.0-beta.1",
+    "react-dom": "^16.5.2",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.0.5",
+    "socket.io-client": "^2.1.1"
+  },
+  "scripts": {
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "eject": "react-app-rewired eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": [
```

Comment:
@TITAN9389 What is this? just looking to learn, appreciate if you can share a link to a tutorial on this (I'm a backend guy... :])

Reply:
the eslintConfig that comes with the CRA built in , or  you can include custom one for more Strict linting.
and browsers list is kind of prefix for css styles.